### ISSUE TITLE
Problem with Installing Nuget

### ISSUE THREAD

**Issue by vamsibulusu** [mono/CppSharp#1065]
I am a beginner and am having a problem with installing Nuget. The InstallNugets.sh script gives the below error and I couldn't find any alternatives to the script. Any help on how to resolve the issue will be appreciated.

OS: Windows 10
Error:

nuget.exe.1                   100%[=================================================>]   1.61M  1.17MB/s    in 1.4s

2018-02-06 02:37:13 (1.17 MB/s) - 'nuget.exe.1' saved [1686528/1686528]

WARNING: Error: TrustFailure (The authentication or decryption has failed.)
Unable to find version '3.6.1' of package 'NUnit.ConsoleRunner'.

**Comment by ddobrev** [comment]
I am afraid I have little idea. You could download the package manually and install the references yourself.

**Comment by vamsibulusu** [comment]
I had no luck with installing manually. Can I get any help on how to use the pre-compiled releases.

**Comment by tritao** [comment]
Just download Nuget from the official website or use it from Visual Studio. That script is not meant to be used from Windows.

**Comment by vamsibulusu** [comment]
Thank you so much. The issue has been resolved. I was actually building from source because I am interested in contributing to CppSharp this summer, under the umbrella of Google Summer of Code. I came  to know about this project from @mohtamohit and now I am prepared with the first thing i.e. I have generated the solution and ran all tests. Feeling good after days of toiling around with this huge repo. :P 

Here I go - 
![image](https://user-images.githubusercontent.com/24713447/35861490-e5b3766a-0b6d-11e8-837e-1860ae844752.png)

Would love to work on a beginner issue now . Anything for me? @ddobrev @tritao
